<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewConfigDialog</class>
 <widget class="QDialog" name="ViewConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VstBoard Appearance</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QListWidget" name="listPalettes"/>
     </item>
     <item>
      <widget class="QListWidget" name="listRoles"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Color</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Red</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="RedSlider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="RedSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Green</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="GreenSlider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="GreenSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Blue</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="BlueSlider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="BlueSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Alpha</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="AlphaSlider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="AlphaSpinBox">
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
         <zorder></zorder>
         <zorder></zorder>
         <zorder></zorder>
         <zorder></zorder>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QCheckBox" name="checkSavedInSetupFile">
       <property name="text">
        <string>Save in setup file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ViewConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ViewConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RedSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RedSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GreenSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GreenSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GreenSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GreenSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BlueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BlueSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BlueSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BlueSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AlphaSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>AlphaSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AlphaSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>AlphaSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RedSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RedSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
